base/applications/regedit/lang/ro-RO.rc

```diff
@@ -33,14 +33,16 @@ Opțiuni:\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /C             Importă conținutul unui fișier de registru.\n\
   /D             Șterge o cheie de registru specificată.\n\
-  /E             Exportă conținutul unei chei de registru specificată într-un fișier.\n\
-                 Dacă nu este nicio cheie specificată, este exportat întregul registru.\n\
+  /E             Exportă conținutul unei chei de registru specificată într-un\n\
+                 fișier. Dacă nu este nicio cheie specificată, este exportat\n\
+				 întregul registru.\n\
   /S             Modul silențios. Nu sunt afișate mesaje.\n\
   /V             Lansează interfața grafică (GUI) în modul avansat. Ignorat.\n\
   /?             Afișează informații și părăsește programul.\n\
-  [filename]     Locația fișierului ce conține informații despre registrul ce urmează\n\
-                 să fie importat. Când e folosit cu [/E], această opțiune specifică\n\
-                 locația fișierului în care informațiile despre registru vor fi exportate.\n\
+  [filename]     Locația fișierului ce conține informații despre registrul ce\n\
+                 urmează să fie importat. Când e folosit cu [/E], această\n\
+                 opțiune specifică locația fișierului în care informațiile\n\
+				 despre registru vor fi exportate.\n\
```

Comment:
Please avoid the use of tab characters. Use only spaces.

Reply:
Ok, I will.